File: Codeforces/1. ICPC Assiut University Community/Contest #2/B. Drawing 'X'/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326907/problem/B

using System;
using static System.Console;

namespace Codeforces.Assuit.Contest2
{
    class ProblemB
    {
        static void Main(string[] args)
        {
            int a = int.Parse(ReadLine());
            for (int i = 0; i < a; i++)
            {
                for (int j = 0; j < a; j++)
                {
                    if (i == (a / 2) && j == i)
                    {
                        Write("X");
                    }
                    else if (j == i)
                    {
                        Write("\\");
                    }
                    else if (j == (a - 1 - i))
                    {
                        Write("/");
                    }
                    else
                    {
                        Write("*");
                    }
                }
                WriteLine();
            }
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Contest #2/B. Drawing 'X'/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326907/problem/B
#include <iostream>
#include <cstdio>

using namespace std;

int main()
{
    int a;
    cin >> a;
    for (int i = 0; i < a; i++)
    {
        for (int j = 0; j < a; j++)
        {
            if (i == (a / 2) && j == i)
            {
                cout << "X";
            }
            else if (j == i)
            {
                cout << "\\";
            }
            else if (j == (a - 1 - i))
            {
                cout << "/";
            }
            else
            {
                cout << "*";
            }
        }
        cout << endl;
    }
}